dspace-server-webapp/src/main/java/org/dspace/app/rest/signposting/converter/LinksetRestMessageConverter.java

```diff
@@ -44,6 +44,9 @@ public static String convert(List<List<LinksetNode>> linksetNodes) {
             if (isNotBlank(linksetNode.getAnchor())) {
                 responseBody.append(format("; anchor=\"%s\" ", linksetNode.getAnchor()));
             }
+            if (isNotBlank(linksetNode.getProfile())) {
+                responseBody.append(format("; profile=\"%s\" ", linksetNode.getProfile()));
+            }
             responseBody.append(", ");
```

Comment:
I know this is not directly related but I have noticed this only now.
It seems like the current signposting implementation is actually leaving a ", " string at the end of the concatenation and this might break the parser for signposting on the other side.
Would you mind checking this refactor this part? I think the following fix should work fine

```
if (responseBody.length() >= 2 && responseBody.substring(responseBody.length() - 2).equals(", ")) {
    responseBody.delete(responseBody.length() - 2, responseBody.length());
}
```